### DISCUSSION TITLE
I made a full tutorial and here my feedback

### DISCUSSION THREAD

**Discussion by FurkanGozukara** [lllyasviel/Fooocus#863] [Show and tell]
Hello @lllyasviel. I made a full tutorial. 

Made a pull request too. It has fully correct manually written Subtitles + video chapters

[**Fooocus Stable Diffusion Web UI - Use SDXL Like You Are Using Midjourney - Easy To Use High Quality**](https://youtu.be/jHTkVm2mcfs)

[![image](https://cdn-uploads.huggingface.co/production/uploads/6345bd89fe134dfd7a0dba40/zuJst_wZ7_qs_2e3KV33W.png)](https://youtu.be/jHTkVm2mcfs)

My feedback is

Please add uncheck all styles
Add option to iterate all styles. like in a foreach loop generate images with each style
add permanent generate option 
add full metadata to the images like automatic1111

**Comment by barepixels** [comment]
1. You can unchek all styles by editing "user_path_config.txt" to look like this
` "default_styles": [],`

2. For iterating all styles I use Fooocus-MRE,  It's something one doesn't do every day.  Since a solution already existed I don't think this is a priority.  Well this is just my opinioin

3. "add permanent generate option" if you mean Generating Forever?  This function exist, right click on Generating Button.

4. "add full metadata to the images like automatic1111" This is my biggest problem with lllyasviel's Fooocus.  Everyone asked for it.  In fact, every other Stabble Diffusion UI have this feature.  For some reason, lllyasviel thinks it's more important to protect newbies from accidentally sharing their recipes.  lllyasviel makes it unnecessarily difficult to document/archive procedures of generating an image we love.  If you ever forget how and unable to recreate an image from the past you will understand the frustration.  Because of this sole reason I don't use lllyasviel's Fooocus as my main production tool.

5. https://www.youtube.com/@OlivioSarikas made a great suggestion.  Being able to swap style.json without having to restart Fooocus

**Comment by FurkanGozukara** [comment]
@barepixels thanks for replies

1 : it should be 1 click icon on the web ui
2 : we need it. it should be a button / option on web ui
3 : dmn i missed this :/ 
4 : yes we need it
5 : i didn't see it :)

**Comment by lllyasviel** [comment]
@FurkanGozukara History generation data can be seen in `Advanced` > `History Log` and you can even use Ctrl+F to search. See also https://github.com/lllyasviel/Fooocus/discussions/160

@barepixels Please stop spamming misinformation that Fooocus cannot track history. You keep spamming this in every post. Also you are violating Policy [699-3](https://github.com/lllyasviel/Fooocus/discussions/699).

Majority users of Fooocus are normal people without programming knowledges or CS expertise. They do not know why image file can hide their inputs, and why should image file hide their inputs. (Many do not know what is PNG or JPEG.) Please give normal non-CS people some normal offline software to generate images. 

This decision is clearly not personal choice: if Dalle3, Midjourney, and FireFly begin to write user history into PNG meta, please let us know and we will have similar things in Fooocus.
